src/vs/workbench/contrib/welcome/page/browser/welcomePage.css

```diff
@@ -3,246 +3,1037 @@
+	align-items: center;
+	text-decoration: none;
+}
+
+.ads_homepage .showOnStartup {
+	margin-top: 10px;
+}
+
+.ads_homepage .link {
+	text-decoration: none;
+}
+
+.ads_homepage .link--learn_more {
+	line-height: 27px;
+}
+
+.ads_homepage .link:hover {
+	text-decoration: underline;
+	cursor: pointer;
+}
+
+.ads_homepage .tile:not(.extension):not(.extension_pack) {
+	border: 1px solid;
+	border-radius: 4px;
+}
+
+.ads_homepage .tile.extension_pack {
+	border: 1px solid;
+	border-radius: 4px;
+}
+
+.ads_homepage .tile {
+	transition: all 0.5s ease;
+	border-radius: 4px;
+}
+
+.ads_homepage .tile:hover:not(.no_hover) {
+	cursor: pointer;
+}
+
+.ads_homepage .link:active {
+	text-decoration: underline;
+}
+
+.ads_homepage .content {
+	padding: 25px 2% 0;
+}
+
+@media only screen and (min-width: 480px) {
+	.ads_homepage .content {
+		padding: 25px 4% 0;
+	}
+}
+
+@media only screen and (min-width: 640px) {
+	.ads_homepage .content {
+		padding: 25px 6% 0;
+	}
+}
+
+@media only screen and (min-width: 1024px) {
+	.ads_homepage .content {
+		padding: 25px 8% 0;
+	}
+}
+
+.ads_homepage .text_container {
+	padding: 15px 15px;
+}
+
+.ads_homepage .gradient {
+	background-size: cover;
+}
+
+@media only screen and (min-width: 480px) {
+	s .ads_homepage .gradient {
+		padding: 25px 4% 0;
+	}
+}
+
+@media only screen and (min-width: 640px) {
+	.ads_homepage .gradient {
+		padding: 25px 6% 0;
+	}
+}
+
+@media only screen and (min-width: 1024px) {
+	.ads_homepage .gradient {
+		padding: 0 8% 0;
+	}
+}
+
+.ads_homepage__section h2 {
+	margin-bottom: 16px;
+}
+
+.ads_homepage__section.hero {
+	background-size: 1128px 467px;
+	background-repeat: no-repeat;
+	background-image: url('../../homeBanner_icon.svg');
+	background-repeat: no-repeat;
+	background-position: 5px 0;
+	background-size: cover;
+	background-repeat: no-repeat;
+}
+
+.vs-dark .ads_homepage__section.hero, .hc-black .ads_homepage__section.hero {
+	background-size: 1128px 467px;
+	background-repeat: no-repeat;
+	background-image: url('../../homeBanner_icon_dark.svg');
+	background-repeat: no-repeat;
+	background-position: 5px 0;
+	background-size: cover;
+	background-repeat: no-repeat;
+}
+
+@media only screen and (min-width: 640px) {
+	.ads_homepage__section.hero {
+		background-size: contain;
+		background-repeat: no-repeat;
+		background-image: url('../../homeBanner_icon.svg');
+		background-position: 300px -54px;
+	}
+}
+
+@media only screen and (min-width: 1024px) {
+	.ads_homepage__section.hero {
+		background-size: 597px;
+		background-repeat: no-repeat;
+		background-image: url('../../homeBanner_icon.svg');
+		background-position: 94% -54px;
+	}
+}
+
+@media only screen and (min-width: 640px) {
+	.vs-dark .ads_homepage__section.hero, .hc-black .ads_homepage__section.hero {
+		background-size: contain;
+		background-repeat: no-repeat;
+		background-image: url('../../homeBanner_icon_dark.svg');
+		background-position: 300px 0px;
+	}
+}
+
+@media only screen and (min-width: 1024px) {
```

Comment:
@Charles-Gagnon this could be something related to your question about min resolution support?